frontend/app/src/components/accounts/AccountBalanceTable.vue

```diff
@@ -55,6 +55,56 @@
       <template v-if="isEth2" #item.ownershipPercentage="{ item }">
         <percentage-display :value="item.ownershipPercentage" />
       </template>
+      <template v-if="isEth" #item.numOfDetectedTokens="{ item }">
+        <div class="d-flex align-center justify-end">
+          <account-detected-tokens-dialog
+            :info="getEthDetectedTokensInfo(item.address)"
+            :disabled="detectingTokens(item.address).value || loading"
+            :loading="detectingTokens(item.address).value"
+            @refresh="fetchDetectedTokens(item.address)"
```

Comment:
Would it make sense to just pass only `loading` + `item.address` to the component and handle the others internally?